analysis/diabetes_analysis.ipynb

```diff
@@ -25,6 +41,32 @@
     "from sklearn.linear_model import LogisticRegression"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "0f464a8d-35d0-4b26-8013-3f6481017712",
+   "metadata": {},
+   "source": [
+    "# Summary"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8377f0b6-b44c-4a63-94aa-d4f7ed1496dc",
+   "metadata": {
+    "jp-MarkdownHeadingCollapsed": true
+   },
+   "source": [
+    "# Introduction\n",
+    "\n",
+    "Diabetes is a serious chronic disease characterized by high levels of glucose in the blood, caused by either insufficient insulin production by the pancreas or the body’s inability to effectively use insulin. It has become a significant global health issue, with its prevalence nearly doubling since 1980, rising from 4.7% to 8.5% in adults by 2014 (World Health Organization, 2016). Early detection enables timely interventions, reduces complications, lowers healthcare costs, and improves quality of life and long-term outcomes (Marshall & Flyvbjerg, 2006).\n",
```

Comment:
"rising from 4.7% to 8.5% in adults by 2014 (World Health Organization, 2016)"

This data seems a bit obsolete considering 10 years have passed. By chance we have access to more recent info?

Would also expect higher rates so that would benefit us in the sense that the importance of our analysis is heightened.

Reply:
updated per suggestion